biomage/account/account.py

```diff
@@ -190,14 +240,35 @@ def create_users_list(user_list, header, input_env):
     with open(user_list + ".out", "w") as out:
         df = pd.read_csv(user_list, header=header, quoting=csv.QUOTE_ALL)
         for _, full_name, email in df.itertuples():
+
+            error = _validate_input(email, full_name)
+            if error:
+                print(error)
+                sys.exit()
+
             password = generate_password()
 
             full_name = full_name.title()
             email = email.lower()
+
+            if not overwrite:
```

Comment:
all of this is not needed, the logic is already handled in line 143 of `account.py` you can add your logs there if needed.